RuntimeUnityEditor.Bepin5/PatchInspector/PatchInspector.cs

```diff
@@ -0,0 +1,1375 @@
+						currentPatches.AddRange(currentPatchInfo.Postfixes.Select(p => p.PatchMethod));
+					if (currentPatchInfo.Transpilers != null)
+						currentPatches.AddRange(currentPatchInfo.Transpilers.Select(p => p.PatchMethod));
+					if (currentPatchInfo.Finalizers != null)
+						currentPatches.AddRange(currentPatchInfo.Finalizers.Select(p => p.PatchMethod));
+				}
+
+				foreach (var patch in storedPatches)
+				{
+					patch.IsEnabled = currentPatches.Contains(patch.PatchMethod as MethodInfo);
+				}
+			}
+			catch
+			{
+				// ignored
+			}
+		}
+
+		private void DrawILViewerWindow(int windowId, ILViewerWindow window)
+		{
+			GUILayout.BeginVertical();
+			GUILayout.Label($"Method: {window.Method.DeclaringType?.FullName}.{window.Method.Name}");
+			GUILayout.Label($"Parameters: {GetMethodParameters(window.Method)}");
+			GUILayout.Label($"Return Type: {GetReturnType(window.Method)}");
+
+			GUILayout.Space(10);
+
+			GUILayout.BeginHorizontal();
+
+			bool opSelected = window.CurrentView == ILViewMode.Original;
+			if (opSelected) GUI.color = Color.yellow;
+			if (GUILayout.Button("Original Method", GUILayout.Height(30)))
+			{
+				window.CurrentView = ILViewMode.Original;
+				window.ScrollPosition = Vector2.zero;
+			}
+			if (opSelected) GUI.color = Color.white;
+
+			bool patchMethodsSelected = window.CurrentView == ILViewMode.PatchMethods;
+			if (patchMethodsSelected) GUI.color = Color.yellow;
+			if (GUILayout.Button($"Patch Manager ({window.PatchMethods.Count})", GUILayout.Height(30)))
+			{
+				window.CurrentView = ILViewMode.PatchMethods;
+				window.ScrollPosition = Vector2.zero;
+			}
+			if (patchMethodsSelected) GUI.color = Color.white;
+
+			GUILayout.FlexibleSpace();
+
+			if (GUILayout.Button("Close", GUILayout.Width(60), GUILayout.Height(30)))
+				window.IsOpen = false;
+
+			GUILayout.EndHorizontal();
+
+			GUILayout.Space(5);
+
+			switch (window.CurrentView)
+			{
+				case ILViewMode.Original:
+					DrawOriginalMethodView(window);
+					break;
+				case ILViewMode.PatchMethods:
+					DrawPatchManagerView(window);
+					break;
+			}
+
+			GUILayout.EndVertical();
+			GUI.DragWindow();
+		}
+
+		private void DrawOriginalMethodView(ILViewerWindow window)
+		{
+			GUILayout.Label("Original Method IL Code:");
+			window.ScrollPosition = GUILayout.BeginScrollView(window.ScrollPosition, GUILayout.Height(450));
+			GUILayout.TextArea(window.OriginalIL);
+			GUILayout.EndScrollView();
+		}
+
+		private void DrawPatchManagerView(ILViewerWindow window)
+		{
+			if (window.PatchMethods.Count == 0)
+			{
+				GUILayout.Label("No patches found for this method.");
+				return;
+			}
+
+			GUILayout.BeginHorizontal();
+			GUILayout.BeginVertical(GUILayout.Width(400));
+			GUILayout.Label("Patches:");
+
+			if (GUILayout.Button("Refresh Patch List", GUILayout.Height(25)))
+				RefreshPatchList(window);
+
+			window.PatchListScrollPosition = GUILayout.BeginScrollView(window.PatchListScrollPosition, GUILayout.Height(400));
```

Comment:
The static heights in here cause this 
<img width="966" height="178" alt="Image" src="https://github.com/user-attachments/assets/30f81b96-ccd2-4eb5-a0f3-e8dd90c8ad85" />